pcsx2/Frontend/HostSettings.cpp

```diff
@@ -250,5 +251,11 @@ void Host::Internal::UpdateEmuFolders()
 					GSTextureReplacements::ReloadReplacementMap();
 			});
 		}
+
+		if (EmuFolders::Videos != old_videos_directory)
+		{
+			GSEndCapture();
```

Comment:
This isn't thread-safe. You should do something like:
```suggestion
if (VMManager::HasValidVM())
{
  GetMTGS().RunOnGSThread([]() {
    GSEndCapture();
  });
}
```